install.sh

```diff
@@ -40,7 +40,12 @@ esac
 
 # This will allow things to work without sudo
 crew_folders="bin cache doc docbook etc include lib lib$LIB_SUFFIX libexec man sbin share tmp var"
-for folder in $crew_folders ; do sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}"/"$folder" ; done
+for folder in $crew_folders
+do
+  if [ -d "${CREW_PREFIX}"/"${folder}" ] || [ -L "${CREW_PREFIX}"/"${folder}" ]; then 
```

Comment:
I don't think we should be changing permissions on symlinks.

Reply:
adjusted